labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -77,7 +98,7 @@ object RecursionPatternMatchingExercise {
    * List(List(1), List('A, 'B, 'C), List('a, 'b)) -> List(List(1, 'A, 'a))
    */
   def zipMultipleWithDifferentSize(in: List[List[_]]): List[List[_]] = {
-    error("fix me")
+    in.map(_.head) :: zipMultiple(in.map(_.tail))
```

Comment:
^

It's also not entirely clear to me whether this will handle multiple sizes correctly.
The reference to `zipMultiple` is also concerning.